### PR TITLE
Add nginx_req_limit_exceeded scenario

### PR DESCRIPTION
**Pull request by sbs2001**
Fixes https://github.com/crowdsecurity/hub/issues/420

### CODE REVIEW

File: scenarios/crowdsecurity/nginx_req_limit_exceeded.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Detects IPs which violate nginx's user set request limit.
```
Comment (buixor) [inline scenarios/crowdsecurity/nginx_req_limit_exceeded.md:1]: Please add info about bucket :)

**Review by buixor — CHANGES_REQUESTED** [review]
imp doc, but overall good :)

File: scenarios/crowdsecurity/nginx_req_limit_exceeded.md (hunk 1 above)
Comment (sbs2001) [reply scenarios/crowdsecurity/nginx_req_limit_exceeded.md:1]: done

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**